app/js/controllers/main.js

```diff
@@ -92,4 +96,36 @@ export default class MainController extends BaseController {
         console.error('Error while subscribing to notifications:', err);
       });
   }
+
+  /**
+   * Will init the `voiced` property, which controls whether a notification is
+   * spoken.
+   *
+   * This works by using a specific hash while loading the application.
+   * Recognized syntaxes are: voiced=1/true/0/false
+   *
+   * Note that we don't change the property right after changing the hash (no
+   * hashchange event handler), so you need to reload the application with the
+   * hash specified.
+   */
+  [p.initVoiced]() {
+    const forcedMatch = location.hash.match(/\bvoiced=(.+?)(&|$)/);
+
+    if (forcedMatch) {
+      const matchedValue = forcedMatch[1];
+      const willSpeak = matchedValue === '1' || matchedValue === 'true';
+
+      console.log('Will force the `voiced` property to ', willSpeak);
+      if (willSpeak) {
+        localStorage['voiced'] = 'true'; // localStorage values are strings
```

Comment:
I'm not sure the best place for settings.js is in server/ especially if we start using it in lib/controllers. How about moving it to common/ ?